rules/windows/builtin/security/win_security_successful_external_remote_smb_login.yml

```
@@ -25,23 +25,18 @@ detection:
         EventID: 4624
         LogonType: 3
     filter_ipv4:
-        SubjectUserName: '-'
         IpAddress|cidr:
-            - 10.0.0.0/8
-            - 172.16.0.0/12
-            - 192.168.0.0/16
-            - 224.0.0.0/4
-            - 127.0.0.0/8
+            - '127.0.0.0/8'
+            - '10.0.0.0/8'
+            - '172.16.0.0/12'
+            - '192.168.0.0/16'
     filter_ipv6:
-        SubjectUserName: '-'
-        IpAddress|startswith:
-            - '::1'  # IPv6 loopback variant
-            - '0:0:0:0:0:0:0:1'  # IPv6 loopback variant
+        - IpAddress: '::1' # IPv6 loopback
+        - IpAddress|startswith:
             - 'fe80:'  # link-local address
-            - 'fc'  # private address range fc00::/7
-            - 'fd'  # private address range fc00::/7
+            - 'fc00:'  # private address range fc00::/7
+            - 'fd00:'  # private address range fd00::/8
```

please revert, it is just wrong as the /7 prefix allows possible subnets e.g. starting with `fdff:`

```suggestion
            - 'fc'  # private address range fc00::/7
            - 'fd'  # private address range fc00::/7
```